<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorDialog</class>
 <widget class="QDialog" name="CalculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluate expression</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="main_layout" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="save_layout">
       <item>
        <widget class="QLabel" name="saved_label">
         <property name="text">
          <string>Saved expressions:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="saved_list"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="list_buttons_layout">
         <item>
          <widget class="QPushButton" name="save_button">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_button">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="evaluation_layout">
       <item>
        <widget class="QLabel" name="statements_label">
         <property name="text">
          <string>Statements:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="statements_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="expression_label">
         <property name="text">
          <string>Expression:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="expression_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="result_label">
         <property name="text">
          <string>Result table:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="result_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
